specification/mobilenetwork/resource-manager/readme.java.md

```diff
@@ -0,0 +1,43 @@
+## Java
+These settings apply only when `--java` is specified on the command line.
+
+``` yaml $(tag) == 'package-2024-02' && $(java)
```

Comment:
$(tag) == 'package-2024-02' seems not needed, we need it to apply to all future tags.

Reply:
Fixed in the new version.